OIPA/common/download_file.py

```diff
@@ -0,0 +1,300 @@
+"""Downloads files from http or ftp locations.
```

Comment:
The initial package has a module to connect to AWS S3 bucket. Some warnings are raised.
So I deleted the S3 module in initial package.